diesel_migrations/migrations_internals/src/connection.rs

```diff
@@ -41,7 +41,7 @@ where
     }
 
     fn latest_run_migration_version(&self) -> QueryResult<Option<String>> {
-        use diesel::dsl::max;
```

Comment:
`diesel::dsl` has re-exported an ambiguous ﻿max, one exported from ﻿`dsl::helper_types::*` and another from ﻿`dsl::expression::dsl`, see: https://github.com/diesel-rs/diesel/blob/1.4.x/diesel/src/lib.rs#L220-L224

Reply:
This pattern continues to exist in diesel 2.0. See https://github.com/diesel-rs/diesel/issues/3745 for some discussion around that. As it is hard to completely remove this bug without doing a breaking change (and therefore a major version bump) what's the impact on that?

Reply:
> This pattern continues to exist in diesel 2.0

The regression test report doesn't indicate any regression in ﻿`diesel@2.0`, so I need to investigate further to determine if it's truly identical to 1.4.x.